manifests/pipelines/git-fetch-pipeline.yaml

```diff
@@ -348,8 +378,14 @@ spec:
     runAfter:
     - retrieve-build-image-info
     taskRef:
-      kind: ClusterTask
-      name: skopeo-copy
+      resolver: git
+      params:
+        - name: url
+          value: https://github.com/tektoncd/catalog.git
+        - name: revision
+          value: main
+        - name: pathInRepo
+          value: task/skopeo-copy/0.3/skopeo-copy.yaml
```

Comment:
I noticed that now it uses images from `quay.io`. Before it used cluster tasks installed by OpenShift Pipelines, which means it used images from `registry.redhat.io`. This may be the reason why `skopeo-copy` task is failing with:
```
level=fatal msg="initializing source docker://image-registry.openshift-image-registry.svc:5000/pipeline-test-tensorflow-housing/tensorflow-housing@sha256:2b6b85d6c3998ae2efb73c931755710bb7ac000925bc6c15479b6341ac3d1bff: reading manifest sha256:2b6b85d6c3998ae2efb73c931755710bb7ac000925bc6c15479b6341ac3d1bff in image-registry.openshift-image-registry.svc:5000/pipeline-test-tensorflow-housing/tensorflow-housing: authentication required"
```
So maybe with Tekton task the authentication is not working correctly with OpenShift Pipelines operator and there may be some incompatibility. I have rerun the PR check to see if it is persistent. We may need to use the tasks directly from RH to ensure the compatibility.